### ISSUE TITLE
Setting uneven Quota

### ISSUE THREAD

**Issue by Zottelchen** [nextcloud/server#25733]
### Steps to reproduce
1. Open User management
2. Set quota to:
2a. 1500 GB
2b. 1,5 TB
2c. 1100 GB
2d. 1,1 TB
2e. 1024 GB
2f. 1024 TB

### Expected behaviour
Quota is:
2a. 1500 GB / 1,5 TB
2b. 1,5 TB
2c. 1100 GB / 1,1 TB
2d. 1,1 TB
2e. 1024 GB
2f. 1024 TB

### Actual behaviour

2a. unlimited quota (+∞)
2b. unlimited quota (+∞)
2c. unlimited quota (+∞)
2d. unlimited quota (+∞)
2e. 1 TB (- 24 GB)
2f. 1024 TB (✔)

## Server configuration detail

**Operating system:** Linux 5.4.0-64-generic #72-Ubuntu SMP Fri Jan 15 10:27:54 UTC 2021 x86_64

**Webserver:** nginx/1.18.0 (fpm-fcgi)

**Database:** mysql 10.3.27

**PHP version:** 

7.4.13
Modules loaded: Core, date, libxml, openssl, pcre, sqlite3, zlib, ctype, curl, dom, fileinfo, filter, ftp, hash, iconv, json, mbstring, SPL, PDO, session, posix, readline, Reflection, standard, SimpleXML, pdo_sqlite, Phar, tokenizer, xml, xmlreader, xmlwriter, mysqlnd, cgi-fcgi, apcu, bcmath, exif, gd, gmp, imagick, intl, ldap, memcached, pcntl, pdo_mysql, pdo_pgsql, redis, sodium, zip, Zend OPcache

**Nextcloud version:** 20.0.4 - 20.0.4.0

**Updated from an older Nextcloud/ownCloud or fresh install:** 

**Where did you install Nextcloud from:** unknown

<details><summary>Signing status</summary>

Array
(
)

</details>

<details><summary>List of activated apps</summary>

```
Enabled:
 - accessibility: 1.6.0
 - activity: 2.13.4
 - admin_audit: 1.10.0
 - bookmarks: 4.1.0
 - breezedark: 20.0.4
 - bruteforcesettings: 2.0.1
 - checksum: 1.0.1
 - cloud_federation_api: 1.3.0
 - comments: 1.10.0
 - contactsinteraction: 1.1.0
 - cookbook: 0.8.1
 - dav: 1.16.2
 - end_to_end_encryption: 1.6.2
 - federatedfilesharing: 1.10.2
 - federation: 1.10.1
 - files: 1.15.0
 - files_accesscontrol: 1.10.2
 - files_pdfviewer: 2.0.1
 - files_rightclick: 0.17.0
 - files_sharing: 1.12.1
 - files_trashbin: 1.10.1
 - files_versions: 1.13.0
 - files_videoplayer: 1.9.0
 - forms: 2.1.0
 - impersonate: 1.7.0
 - issuetemplate: 0.7.0
 - logreader: 2.5.0
 - lookup_server_connector: 1.8.0
 - nextcloud_announcements: 1.9.0
 - notifications: 2.8.0
 - oauth2: 1.8.0
 - password_policy: 1.10.1
 - photos: 1.2.1
 - privacy: 1.4.0
 - provisioning_api: 1.10.0
 - quota_warning: 1.9.1
 - serverinfo: 1.10.0
 - settings: 1.2.0
 - survey_client: 1.8.0
 - suspicious_login: 3.2.1
 - systemtags: 1.10.0
 - text: 3.1.0
 - theming: 1.11.0
 - twofactor_backupcodes: 1.9.0
 - twofactor_totp: 5.0.0
 - updatenotification: 1.10.0
 - viewer: 1.4.0
 - workflowengine: 2.2.0
Disabled:
 - dashboard
 - encryption
 - firstrunwizard
 - recommendations
 - sharebymail
 - support
 - user_ldap
 - user_status
 - weather_status

```
</details>

<details><summary>Configuration (config/config.php)</summary>

```
{
    "skeletondirectory": "\/var\/www\/html\/themes\/hetzner\/skeleton",
    "memcache.local": "\\OC\\Memcache\\APCu",
    "apps_paths": [
        {
            "path": "\/var\/www\/html\/apps",
            "url": "\/apps",
            "writable": false
        },
        {
            "path": "\/var\/www\/html\/custom_apps",
            "url": "\/custom_apps",
            "writable": true
        }
    ],
    "passwordsalt": "***REMOVED SENSITIVE VALUE***",
    "secret": "***REMOVED SENSITIVE VALUE***",
    "datadirectory": "***REMOVED SENSITIVE VALUE***",
    "dbtype": "mysql",
    "version": "20.0.4.0",
    "overwrite.cli.url": "https:\/\/nx7238.your-storageshare.de",
    "dbname": "***REMOVED SENSITIVE VALUE***",
    "dbhost": "***REMOVED SENSITIVE VALUE***",
    "dbport": "",
    "dbtableprefix": "oc_",
    "mysql.utf8mb4": true,
    "dbuser": "***REMOVED SENSITIVE VALUE***",
    "dbpassword": "***REMOVED SENSITIVE VALUE***",
    "installed": true,
    "instanceid": "***REMOVED SENSITIVE VALUE***",
    "redis": {
        "host": "***REMOVED SENSITIVE VALUE***",
        "port": "6379",
        "timeout": "1.5",
        "dbindex": "0"
    },
    "filelocking.enabled": true,
    "memcache.locking": "\\OC\\Memcache\\Redis",
    "mail_smtpmode": "smtp",
    "mail_smtpauthtype": "LOGIN",
    "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
    "mail_from_address": "***REMOVED SENSITIVE VALUE***",
    "mail_domain": "***REMOVED SENSITIVE VALUE***",
    "log_type": "syslog",
    "syslog_tag": "nextcloud-nx7238",
    "simpleSignUpLink.shown": false,
    "upgrade.disable-web": true,
    "updatechecker": false,
    "theme": "hetzner",
    "loglevel": 2,
    "maintenance": false,
    "trusted_domains": [
        "nx7238.your-storageshare.de",
    ]
}
```
</details>

**Are you using external storage, if yes which one:** no

**Are you using encryption:**  no

**Are you using an external user-backend, if yes which one:** no

## Client configuration

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.182 Safari/537.36

**Operating system:** Windows 10

## Logs

<details><summary>Web server error log</summary>

```
No access
```
</details>

<details><summary>Nextcloud log</summary>

```
No access
```
</details>

<details><summary>Browser log</summary>

Not relevant
</details>

**Comment by kesselb** [comment]
Thanks for reporting :+1: 

Closing in favor of https://github.com/nextcloud/server/issues/24002

**Comment by rubo77** [comment]
#24002 is closed in favor to #18468

workaround: 

    occ user:setting USERNAME files quota 1.5GB
